scalardb-test/src/main/java/kelpie/scalardb/transfer/TransferCommon.java

```diff
@@ -44,15 +44,17 @@ public static DistributedStorage getStorage(Config config) {
     return storage;
   }
 
-  public static Get prepareGet(int id, int type) {
-    Key partitionKey = new Key(ACCOUNT_ID, id);
+  public static Get prepareGet(Config config, int id, int type) {
+    String accountPrefix = config.getUserString("test_config", "account_prefix");
+    Key partitionKey = new Key(ACCOUNT_ID, accountPrefix + "-" + String.valueOf(id));
```

Comment:
Can you clarify why the account prefix is needed?

Reply:
I apologize for the inconvenience :bow:

Added support for specifying `account prefix` to support use cases where you want to run processes in parallel.

Reply:
I think you can run it in parallel with multiple processes without the prefix.
What kind of issue did you face?

Reply:
A different process runs the process in the same way, but the preprocess also works in the same way. However, since it refers to the same DB, the id is duplicated and an error occurs.

Reply:
We will use a different method.

Reply:
Ah, that's because the data loading part is also done in parallel with multiple processes. (The current data loading code is designed for multiple threads but not for multiple processes since it's trying to create initial records if they don't exist.)
The process phase should be fine with multiple threads or multiple processes. (going to the same accounts at the same time is just treated as a conflict.)

So, the preparation phase (loading phase) should be done by one process (with multiple threads).

Reply:
I understood. I'll fix it.

Reply:
This has been handled by making the registration process only one process. :bow: